src/demo-app/live-announcer/live-announcer-demo.ts

```diff
@@ -19,6 +19,6 @@ export class LiveAnnouncerDemo {
   constructor(private live: LiveAnnouncer) {}
 
   announceText(message: string) {
-    this.live.announce(message);
+    this.live.announce(message, undefined, 1500);
```

Comment:
Should we try having a method overload? That way we can have a nicer API.

```ts
this.live.announce(message, 1500)
this.live.announce(message);
this.live.announce(message, 'polite', 1500);
this.live.announce(message, 'polite');
```

We did the same for the `MatRipple` launch method.